src/app/style.less

```diff
@@ -50,3 +50,12 @@ body {
     }
   }
 }
+
+.ant-drawer-body {
+  height: calc(100vh - @drawer-header-height);
```

Comment:
This allows the drawer to be full height so that the password disclaimer is positioned at the bottom.